### PR TITLE
Redesign all pages

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-7057, L2B-7056, L2B-7153, L2B-7151, L2B-7152, L2B-7148, L2B-7149, L2B-7071, L2B-7150, L2B-7024, L2B-7548

### CODE REVIEW

File: packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts [hunk 1]
```diff
@@ -38,7 +47,7 @@ export async function getScalingSummaryEntries() {
         : undefined
 
     const common = {
-      entryType: 'scaling' as const,
+      entryType: 'scalingV2' as const,
```
Comment (imxeno) [inline packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts:50]: Should this still be scalingV2?

File: packages/frontend/src/components/badge/stage-badge.tsx [hunk 2]
```diff
@@ -4,54 +4,42 @@ import { cn } from '~/utils/cn'
...
+      return 'bg-[#FFC61B] border border-[#D9AA1E] text-yellow-900'
     case 'Stage 0':
-      return 'bg-orange-400 text-black'
+      return 'bg-orange-600 border border-[#F98A24] dark:bg-[#E03109] dark:border-[#F25430] text-white'
     case 'UnderReview':
-      return 'bg-zinc-700 text-yellow-200'
+      return 'border border-current text-n-yellow-700 dark:text-yellow-200'
     case 'NotApplicable':
-      return 'bg-gray-200 dark:bg-zinc-700 text-gray-500 dark:text-gray-400 !font-normal'
+      return 'bg-[#B4C7D5] dark:bg-[#3D4361] border border-[#D1E1EC] dark:border-[#343955] text-zinc-800 dark:text-primary  '
```
Comment (imxeno) [inline packages/frontend/src/components/badge/stage-badge.tsx:42]:
```suggestion
      return 'bg-[#B4C7D5] dark:bg-[#3D4361] border border-[#D1E1EC] dark:border-[#343955] text-zinc-800 dark:text-primary'
```

File: packages/frontend/src/app/(side-nav)/scaling/finality/page.tsx [hunk 3]
```diff
@@ -18,10 +17,12 @@ export default async function Page() {
...
     <ScalingFilterContextProvider>
-      <SimplePageHeader>Finality</SimplePageHeader>
+      <MainPageHeader>Finality</MainPageHeader>
       <FinalityWarning />
-      <ScalingFinalityTable projects={projects} />
-      <FinalityDiagramsSection className="mt-20" diagrams={finalityDiagrams} />
+      <MainPageCard>
+        <ScalingFinalityTable projects={projects} />
+      </MainPageCard>
+      {/* <FinalityDiagramsSection className="mt-20" diagrams={finalityDiagrams} /> */}
```
Comment (imxeno) [inline packages/frontend/src/app/(side-nav)/scaling/finality/page.tsx:25]: 👀

File: packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/server/features/scaling/summary/get-scaling-summary-entries.ts:50]: nope

File: packages/frontend/src/app/(side-nav)/scaling/finality/page.tsx (hunk 3 above)
Comment (torztomasz) [reply packages/frontend/src/app/(side-nav)/scaling/finality/page.tsx:25]: Intended

**Review by imxeno — APPROVED** [review]

**Review by imxeno — APPROVED** [review]

**Pull request merged**